doc/classes/ResourceLoader.xml

```diff
@@ -138,13 +138,19 @@
 			The resource was loaded successfully and can be accessed via [method load_threaded_get].
 		</constant>
 		<constant name="CACHE_MODE_IGNORE" value="0" enum="CacheMode">
-			The resource is always loaded from disk, even if a cache entry exists for its path, and the newly loaded copy will not be cached. Instances loaded with this mode will exist independently.
+			Resource and subresources are neither retrieved from nor stored into the resource cache. External resources are loaded with [constant CACHE_MODE_REUSE].
```

Comment:
Mmmh, more to sink my teeth into. Yummy.
It's weird. The current descriptions in this PR are fairly hard to follow. Also worth noting that the class reference often mentions "from disk" when talking about non-volatile memory.

I do have a suggestion but as I am looking at both old descriptions and this PR's behavior I'm not sure how to word it better at the moment.

```suggestion
The resource and its subresources are [b]always[/b] loaded and stored to disk, and are not cached. A resource from a file is loaded from cache when possible. Otherwise, it is loaded from disk and cached. 
```

Reply:
I'm open-arms to better-worded descriptions. English is, as one can notice, not my mother tongue. Also, I was too focused on making them technically accurate that I may have not written the friendliest descriptions. That said, this suggestion of yours contains a mistake, when you say _stored to disk_. The resources are loaded _from disk_ and stored _(in)to cache_.